app/src/androidTest/resources/jacoco-agent.properties

```diff
@@ -0,0 +1 @@
+output=none
```

Comment:
What is this for?

Reply:
This is for jacoco... It would failed on my machine so I found this fix on github.